src/azul/azulclient.py

```diff
@@ -56,11 +58,9 @@ def post_bundle(self, bundle_fqid, indexer_url):
         Send a mock DSS notification to the indexer
         """
         simulated_event = self._make_notification(bundle_fqid)
-        body = json.dumps(simulated_event).encode('utf-8')
-        request = Request(indexer_url, body)
-        request.add_header("content-type", "application/json")
-        with urlopen(request) as f:
-            return f.read()
+        response = requests.post(indexer_url, json=simulated_event, auth=hmac.prepare())
+        response.raise_for_status()
+        return response
```

Comment:
Shouldn't this be 

```suggestion
        return response.content
```

to be equivalent to the previous implementation?